compiler/qsc_data_structures/src/namespaces/tests.rs

```diff
@@ -0,0 +1,566 @@
+use expect_test::expect;
+
+use super::*;
+
+#[allow(clippy::too_many_lines)]
```

Comment:
Looks like you'll need to add `#![allow(clippy::needless_raw_string_hashes)]` to make clippy happy with this file.